vue/src/client/models/RegionList.ts

```diff
@@ -4,5 +4,5 @@ export type RegionList = {
   count: number;
   next: null;
   previous: null;
```

Comment:
are these still valid with the new pagination?  I forgot on the last PR to check the modelruns.  I know we don't ever have more than one page, but there are no null/string values in the response for this.  Probably should update performers as well (I didn't check that last time).